pkg/dataplane/http/context.go

```diff
@@ -1218,7 +1218,7 @@ func (c *context) encodeTypedAttributes(attributes map[string]interface{}) (map[
 		typedAttributes[attributeName] = make(map[string]interface{})
 		switch value := attributeValue.(type) {
 		default:
-			return nil, fmt.Errorf("unexpected attribute type for %s: %T", attributeName, reflect.TypeOf(attributeValue))
+			return nil, fmt.Errorf("unexpected attribute type for %s: %T. Underlying type %v", attributeName, reflect.TypeOf(attributeValue), value)
```

Comment:
Hm, isn't the correct fix
```suggestion
			return nil, fmt.Errorf("unexpected attribute type for %s: %T", attributeName, value)
```
?

Reply:
Not according to ChatGPT :)

Reply:
lol well [it's wrong](https://go.dev/play/p/KucTHypBzet). Shocking, I know.

Reply:
Hmm, what about https://go.dev/play/p/UERUHwNkCkr ?

Reply:
It doesn't make any sense. It sets the value to the type, and then gets the type of the type in 2 places.

Reply:
As usual its easier to agree with you :)